src/ILCompiler.Compiler/src/Compiler/WindowsNodeMangler.cs

```diff
@@ -63,5 +63,20 @@ public override string MethodGenericDictionary(MethodDesc method)
         {
             return GenericDictionaryNamePrefix + NameMangler.GetMangledMethodName(method);
         }
+
+        public override string ExternMethod(string unmangledName, MethodSignature signature)
+        {
+            TargetDetails target = signature.Context.Target;
+            if (target.Architecture != TargetArchitecture.X86)
+                return unmangledName;
+
+            int signatureBytes = 0;
```

Comment:
I couldn't find the "official algorithm" to compute the number and I'm not sure this is right.

Reply:
The number is the size of the stack arguments that the method takes. It is the same number as N in `ret N` instruction that the stdcall methods end with on x86, except for method with return buffer arguments. The return buffer argument size is not included in this number.

Also, this should work against the unmanaged signature (ie use the argument sizes after marshaling).